From 540deeae3033060f959f9b8a6d2ebfd7b6fa2553 Mon Sep 17 00:00:00 2001
From: ragnarov <101@mailu.io>
Date: Sun, 12 Feb 2023 19:33:08 +0600
Subject: [PATCH] 	modified:   st.c

---
 st.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/st.c b/st.c
index 62def59..5b93cf5 100644
--- a/st.c
+++ b/st.c
@@ -2654,3 +2654,5 @@ redraw(void)
 	tfulldirt();
 	draw();
 }
+
+#include "plugins.h"
-- 
2.39.1

